src/MoBi.Presentation/UICommand/ImportDataRepositoryUICommand.cs

```diff
@@ -36,6 +37,22 @@ public void Execute()
       }
    }
 
+   internal class RemoveSimulationResultCommand : ObjectUICommand<DataRepository>
```

Comment:
I know there are some UI commands without it, but I would prefer a UI command has a name like `RemoveSimulationResultUICommand`. To clarify that it's a UI command when your auto-complete is activated.